src/Servers/HttpSys/src/AsyncAcceptContext.cs

```diff
@@ -127,7 +127,9 @@ internal uint QueueBeginGetContext()
                 statusCode = HttpApi.HttpReceiveHttpRequest(
                     Server.RequestQueue.Handle,
                     _nativeRequestContext.RequestId,
-                    (uint)HttpApiTypes.HTTP_FLAGS.HTTP_RECEIVE_REQUEST_FLAG_COPY_BODY,
```

Comment:
We're making this change to allow request delegation? Will this negatively impact performance? Is so how much?

Reply:
I haven't measured, but it's my understanding that in the worst case (when headers+body) happen to fit in a single TCP packet, you'll need an extra call to Http.Sys to schedule another IOCP callback.

@Tratcher ?

Reply:
In the best case today you get some or all of the request body data the request struct returned from HttpReceiveHttpRequest. You still have to do another native call to verify that was the end of the body.

After this change I think you'd need to do at least two native request body reads, one for the data and one for the end of body check, though if the data were already available they could complete synchronously.

@JamesNK there are Http.Sys gRPC benchmarks with request bodies, right?

Reply:
Worst case, we can always add to the `HttpSysOptions.RequestQueueMode` enum that requires you to configure your queue for delegation ahead of time and avoid opting everyone into the new behavior

Reply:
Find a benchmark to run first. I don't expect there to be a difference.

Reply:
Looks like there's no existing benchmark. I'll ping one of y'all offline for help in setting up a run

Reply:
No existing benchmark why am surprised 😬

Reply:
Why don't we add a super low level option to the API to turn this back on just in case we regress performance.

Reply:
Based my on benchmark, I'm going to resolve this conversation with no new API required